.github/workflows/ci_api.yml

```diff
@@ -13,18 +13,21 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Set up Python 3.7
       uses: actions/setup-python@v1
       with:
         python-version: 3.7
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install black
+        pip install -r ./api/requirements-dev.txt
     - name: Check formatting with black
       run: |
         black --line-length 80 --target-version py37 --exclude migrations --check ./api
+    - name: Run pytest
+      run: |
+        pytest
```

Comment:
I'm still not sure if it's better to have tests run together with formatting
I feel that leaving them separate would be better
- we won't need to worry that one is holding up the other 
- we can have better customizability if we want to run them under different constraints
- we're going to see one failure and no success if either of them fails

Reply:
You are right. We could keep the same file and run parallel jobs I think.